### PR TITLE
Changelog + gooncode disk fix.

### LINKED ISSUE #1946: Gooncode disk has a rogue pixel in the lower left corner.
Picky picky picky.

### PR DESCRIPTION
**Pull request by Kokojos**
Fixes #1946 + adds changelog for the last three PR

### CODE REVIEW

**Comment by Nokington** [comment]
:+1:

**Comment by Nebyab** [comment]
why not

File: html/changelogs/kokofix.yml [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+delete-after: True
+
+# Any changes you've made.  See valid prefix list above.
+# INDENT WITH TWO SPACES.  NOT TABS.  SPACES.
+# SCREW THIS UP AND IT WON'T WORK.
+# Also, this gets changed to [] after reading.  Just remove the brackets when you add new shit.
+# Please surround your changes in  double quotes ("). It works without them, but if you use certain characters it screws up compiling. The quotes will not show up in the changelog.
+changes: 
+  - rscadd: "Fix an icon error with the gooncode disk"
+  - bugfix: "Fix lings regenerating and still being husked."
```
Comment (nalarmg) [inline html/changelogs/kokofix.yml:37]: Is this actually in this PR?

Comment (Nokington) [inline html/changelogs/kokofix.yml:37]: He said it's a changelog for his other prs too

**Comment by Kokojos** [comment]
Closing because changelogs should be in their own PR

**Comment by Pyko1** [comment]
err

**Comment by Pyko1** [comment]
already fixed the disk

**Pull request closed without merging**